spec/controllers/account_reset/cancel_controller_spec.rb

```diff
@@ -25,5 +30,26 @@
       post :cancel
       expect(response).to redirect_to root_url
     end
+
+    it 'sends an SMS' do
+      AccountResetService.new(user).create_request
+      allow(SmsAccountResetCancelJob).to receive(:perform_now)
+
+      post :cancel, params: { token: AccountResetRequest.all[0].request_token }
+
+      expect(SmsAccountResetCancelJob).to have_received(:perform_now).with(
+        phone: user.phone
+      )
+    end
+
+    it 'sends an email' do
+      AccountResetService.new(user).create_request
+
+      @mailer = instance_double(ActionMailer::MessageDelivery, deliver_later: true)
+      allow(UserMailer).to receive(:account_reset_cancel).with(user.email).
+        and_return(@mailer)
+
+      post :cancel, params: { token: AccountResetRequest.all[0].request_token }
+    end
```

Comment:
Can we please add a spec for when the user doesn't have a phone to make sure it doesn't send an SMS?

Reply:
I added a spec to the cancel_controller.  This sms job will not be called if there is not a phone.